utilities/logic/async_wrapper.py

```diff
@@ -115,6 +115,18 @@ def _filter_non_json_lines(data):
 
     return ('\n'.join(lines), warnings)
 
+
+def _get_interpreter(module_path):
+    module_fd = open(module_path, 'rb')
+    try:
+        head = module_fd.read(1024)
+        if head[0:2] != '#!':
+            return None
+        return head[2:head.index('\n')].split(' ', 1)
```

Comment:
should probably add a `strip()` to make this handle leading spaces, since they are technically legal (I'm guessing that'd be handled transparently by the popen, but maybe not)